doc/docs.md

```diff
@@ -5627,30 +5629,46 @@ fn main() {
 	a := Vec{2, 3}
 	b := Vec{4, 5}
 	mut c := Vec{1, 2}
+
 	println(a + b) // "{6, 8}"
 	println(a - b) // "{-2, -2}"
 	c += a
+	//^^ autogenerated from + overload
 	println(c) // "{3, 5}"
 }
 ```
 
-Operator overloading goes against V's philosophy of simplicity and predictability.
-But since scientific and graphical applications are among V's domains,
-operator overloading is an important feature to have in order to improve readability:
+> Operator overloading goes against V's philosophy of simplicity and predictability.
+> But since scientific and graphical applications are among V's domains,
+> operator overloading is an important feature to have in order to improve readability:
+>
+> `a.add(b).add(c.mul(d))` is a lot less readable than `a + b + c * d`.
+
+Operator overloading is possible for the following binary operators: `+, -, *, /, %, <, ==`.
```

Comment:
```suggestion
Operator overloading is possible for the following operators: `+, -, *, /, %, <, ==`.
```
If `binary` is part of the statement, people may immediately wonder "Wait, does this only work with binary numbers, but not normal ints?"

Reply:
if a person thinks so, then this means that he is very beginner and operator overloading in general will be a difficult topic for him.

The concept of a binary operation is the basis of programming; and I see no point in simplifying the sentence.

Moreover, without this word, it would seem that it is possible to overload unary operators, although this is not the case.

https://www.techopedia.com/definition/23953/binary-operator

Reply:
Yes, I know this.  I was thinking of the beginner.  The person who doesn't know the difference.

Reply:
I think if someone does not know, then a quick googling will help him understand what a binary operator is. To hide this important part just so that the user does not get confused seems strange to me.

Reply:
Interesting outlook... while it is more technically correct to say "binary operator", does it really add anything to the documentation?  `+` is still the addition operator no matter if you name it a `binary operator` or just `operator`.

Those who know the correct term will already know.  Those who don't won't be confused if it isn't there.

Reply:
I agree with @i582 here. The phrase `binary operators` here, is clearly applied not to the number, but to the operations, which are listed right after that same phrase.

The terms `unary` and `binary` do clarify, what is meant by something like `+` and `-`, since they can be both.